client/src/components/overview/ImageGallery.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import Thumbnails from './Thumbnails.jsx';
+import Atelier from '/Users/julianzthong/Desktop/Hack_Reactor/FEC/client/src/Atelier.js';
+
+
+function ImageGallery (props) {
+  const [viewState, setView] = useState('default');
+  const [thumbnailList, setThumbnail] = useState(props)
+  // const [productId, setId] = useState(props.productId);
+
+  return (
+    <div
+    onClick={() => setView( 'expanded' )}
```

Comment:
Should this check if the view is already expanded and set it to default if it is?

Reply:
Have not worked on the Image gallery at all other than creating some basic hooks to experiment with. Will use your logic instead